package.json

```diff
@@ -1,67 +1,64 @@
 {
     "name": "artemis",
-    "version": "5.1.6",
+    "version": "5.2.0",
     "description": "Interactive Learning with Individual Feedback",
     "private": true,
     "license": "MIT",
     "cacheDirectories": [
         "node_modules"
     ],
     "dependencies": {
-        "@angular/animations": "12.2.0",
-        "@angular/common": "12.2.0",
-        "@angular/compiler": "12.2.0",
-        "@angular/core": "12.2.0",
-        "@angular/forms": "12.2.0",
-        "@angular/localize": "12.2.0",
-        "@angular/platform-browser": "12.2.0",
-        "@angular/platform-browser-dynamic": "12.2.0",
-        "@angular/router": "12.2.0",
-        "@ctrl/ngx-emoji-mart": "^6.0.0",
+        "@angular/animations": "12.2.6",
+        "@angular/common": "12.2.6",
+        "@angular/compiler": "12.2.6",
+        "@angular/core": "12.2.6",
+        "@angular/forms": "12.2.6",
+        "@angular/localize": "12.2.6",
+        "@angular/platform-browser": "12.2.6",
+        "@angular/platform-browser-dynamic": "12.2.6",
+        "@angular/router": "12.2.6",
+        "@ctrl/ngx-emoji-mart": "6.0.1",
         "@fingerprintjs/fingerprintjs": "3.3.0",
         "@fortawesome/angular-fontawesome": "0.9.0",
         "@fortawesome/fontawesome-svg-core": "1.2.36",
         "@fortawesome/free-regular-svg-icons": "5.15.4",
         "@fortawesome/free-solid-svg-icons": "5.15.4",
         "@ls1intum/apollon": "2.8.0",
-        "@ng-bootstrap/ng-bootstrap": "11.0.0-beta.1",
+        "@ng-bootstrap/ng-bootstrap": "11.0.0-beta.2",
         "@ngx-translate/core": "13.0.0",
         "@ngx-translate/http-loader": "6.0.0",
-        "@popperjs/core": "2.9.3",
-        "@sentry/browser": "6.11.0",
-        "@swimlane/ngx-datatable": "19.0.0",
+        "@popperjs/core": "2.10.1",
+        "@sentry/browser": "6.12.0",
+        "@swimlane/ngx-datatable": "20.0.0",
         "ace-builds": "1.4.12",
         "angular-fittext": "2.1.1",
         "blob-util": "2.0.2",
-        "bootstrap": "5.1.0",
+        "bootstrap": "5.1.1",
         "chart.js": "2.9.4",
         "compare-versions": "3.6.0",
-        "core-js": "3.16.0",
-        "crypto-browserify": "3.12.0",
+        "core-js": "3.18.0",
+        "crypto-js": "4.1.1",
         "css-minimizer-webpack-plugin": "3.0.2",
+        "dayjs": "1.10.7",
         "diff-match-patch-typescript": "1.0.8",
-        "dompurify": "2.3.0",
+        "dompurify": "2.3.2",
         "export-to-csv": "0.2.1",
-        "franc-min": "5.0.0",
+        "franc-min": "6.0.0",
         "interactjs": "1.10.11",
         "jquery": "3.6.0",
         "js-video-url-parser": "0.4.3",
-        "lodash": "4.17.21",
-        "lodash.template": "4.5.0",
+        "lodash-es": "4.17.21",
```

Comment:
Any idea why the tests apparently work now with that? Since previously our issue with ng2-charts was that it was loading `lodash-es` and it took hours.